### DISCUSSION TITLE
Welcome to Symja Discussions!

### DISCUSSION THREAD

**Discussion by axkr** [axkr/symja_android_library#205] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

 If you want to chat about the further development of the Symja project please: <a href="https://discord.gg/tYknzr2qam" target="blank"><img src="https://img.shields.io/discord/869895703718166529.svg?color=7289da&label=Join our Discord&logo=discord&style=for-the-badge" alt="Join our Discord" />

**Comment by Celestin-Dev** [comment]
Comment integrer symja dans une projet android studio

**Comment by axkr** [comment]
There is the example project which shows the basics how to integrate Symja into an Android project:
- https://github.com/axkr/symja-example

**Comment by Celestin-Dev** [comment]
Hello,,
I am new for using a librairies symja in android. You can help me for using
it in android
?


Le mar. 15 juil. 2025 à 18:42, Axel Kramer ***@***.***> a
écrit :

**Reply by axkr** [reply]
> Hello,, I am new for using a librairies symja in android. You can help me for using it in android ?
- Whats your developer background? 
- Did you already created some Android apps?
- What Android app do you like to create from Symja?
- What did you already try?

**Reply by Celestin-Dev** [reply]
I'd like to create an application for symplify a maths complex writing to simple writing
